frontend/src/pages/org/browser-profiles-detail.ts

```diff
@@ -679,13 +679,26 @@ export class BrowserProfilesDetail extends BtrixElement {
     };
 
     try {
-      const data = await this.api.fetch<{ updated: boolean }>(
-        `/orgs/${this.orgId}/profiles/${this.profileId}`,
-        {
-          method: "PATCH",
-          body: JSON.stringify(params),
-        },
-      );
+      let data;
+
+      // eslint-disable-next-line no-constant-condition, @typescript-eslint/no-unnecessary-condition
```

Comment:
Instead of disabling the next line we could set a maximum number of retries, something like `while(retries < 100)`